src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+  .stars--1.stars__star:nth-child(1),
+  .stars--1.stars__star:nth-child(-n+2),
+  .stars--1.stars__star:nth-child(-n+3),
+  .stars--1.stars__star:nth-child(-n+4),
+  .stars--1.stars__star:nth-child(-n+5) {
```

Comment:
Everything for stars--`1`? Try use [cycle](https://sass-lang.com/documentation/at-rules/control/for)